src/server/model/image.js

```diff
@@ -14,6 +14,8 @@ const imageSchema = new Schema({
   preview: {type:String},
   fullSize: {type: String},
 
+  isGalleryThumbnail: {type: Boolean, default: false},
```

Comment:
Why not a separate table for thumbnail images to avoid pot. ambiguity in the images table?

I can easily imagine a scenario where somebody fetches all the images and end up with weirdly sized thumbnails in the mix, and possibly duplicates, by doing it this way :/ You should at least add some kind of comment to make this less ambiguous imo.

PS. [boolean blindness is a thing](https://existentialtype.wordpress.com/2011/03/15/boolean-blindness/)